xml/net_dhcp.xml

```diff
@@ -482,13 +482,21 @@
    ISC) and tools coming with the <systemitem>wicked</systemitem> package.
   </para>
 
+   <note>
+    <title>KEA DHCP replaces ISC DHCP Server</title>
+    <para>
+     The Internet Systems Consortium (ISC) has developed a new DHCP server, KEA
+     which replaces ISC DHCP servers. For more information, refer to <link xlink:href="https://www.isc.org/kea/"/>
+    </para>
+   </note>
+
   <para>
    By default, the <systemitem>wicked</systemitem> tools are installed with the
    services <systemitem>wickedd-dhcp4</systemitem> and
    <systemitem>wickedd-dhcp6</systemitem>. Both are launched automatically on
    each system boot to watch for a DHCP server. They do not need a
    configuration file to do their job and work out of the box in most standard
-   setups. For more complex situations, use the ISC
+   setups. For more complex situations, use the KEA
```

Comment:
Kea doesn't have a dhcp client. For SLE16 it is replaced with [dhcpcd](https://github.com/NetworkConfiguration/dhcpcd) and for SLE15 for now still only the ISC dhclient is available.